SpatialGDK/Source/SpatialGDK/Public/Schema/Interest.h

```diff
@@ -118,7 +118,7 @@ struct Query
 
 	// Either full_snapshot_result or a list of result_component_id should be provided. Providing both is invalid.
 	TSchemaOption<bool> FullSnapshotResult; // Whether all components should be included or none.
-	SchemaResultType ResultComponentIds;	// Which components should be included.
```

Comment:
You can still ask for individual result component ids - might be useful to maintain the functionality.

Reply:
It would mean we don't have to wrap things like query tags in component sets also